src/components/actionSheet/index.js

```diff
@@ -134,12 +155,15 @@ export default class ActionSheet extends BaseComponent {
   }
 
   renderActions() {
-    const {title, options, cancelButtonIndex} = this.props;
+    const {title, options, cancelButtonIndex, renderAction, listStyle} = this.props;
     const optionsToRender = _.filter(options, (option, index) => index !== cancelButtonIndex);
     
     return (
-      <View paddingB-8 paddingT-8={_.isEmpty(title)}>
-        {_.map(optionsToRender, this.renderAction)}
+      <View style={[_.isEmpty(title) ? styles.listNoTitle : styles.listWithTitle, listStyle]}>
+        {renderAction ?
```

Comment:
You are not checking if renderAction is a function before you call it?

Reply:
Done